backend/app/views/spree/admin/shared/_sidebar_left.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="sidebar">
+  <%= render partial: 'spree/admin/shared/sidebar_left_header' %>
```

Comment:
There's currently a thing called "sidebar" which refers to the right content of a page with a 3rd level nav (e.g., /admin/general_settings/edit).